.gitignore

```diff
@@ -16,4 +16,5 @@
 .factorypath
 .vscode/
 repo/
-/*.svg
\ No newline at end of file
+/*.svg
+/**/.cache
```

Comment:
Where is that needed and why would we ignore any `.cache` subfolder at any level of the tree?

Reply:
yes should be specific, but there is not collision so its not an big deal actually.